membership-attribute-service/app/services/AttributesMaker.scala

```diff
@@ -50,6 +50,32 @@ class AttributesMaker extends LazyLogging {
     }
   }
 
+  def attributesAndSource(zuoraAttributes: Option[ZuoraAttributes], dynamoAttributes: Option[DynamoAttributes]): (String, Option[Attributes]) = {
```

Comment:
It's a shame that we need this at all now that we have our ZuoraAttributes and DynamoAttributes classes... but I guess we can't get rid of the more generic Attributes until we remove MembershipNumber and AdFree from the responses, and that's one for another PR!